VirtualTest/Test2/Test2.ino

```diff
@@ -0,0 +1,360 @@
+#include "Tile.h"
```

Comment:
I'm confused about this file and the other virtual one, both look very similar.

Reply:
What is the purpose of this file?

Reply:
I'm sorry, I don't understand. Which virtual file are you talking about?

Reply:
The purpose of Test2.ino is to test all the virtual functions and verify they are working correctly.